pkg/rancher-desktop/components/Preferences/BodyKubernetes.vue

```diff
@@ -4,6 +4,7 @@ import { Banner } from '@rancher/components';
 import Vue from 'vue';
 import { mapGetters } from 'vuex';
 
+import { firstStableVersion } from '@pkg/backend/k3sHelper';
```

Comment:
Hmm, `k3sHelper` imports a bunch of NodeJS things (so it probably expects to be run from the main process); but I guess if it works it's probably fine? It's odd to import a `backend` file in the front end, though.